src/_includes/partials/helpus.njk

```diff
@@ -0,0 +1,33 @@
+<hr />
+
+<h2 class="mt-500 mb-400">
+  {{ helpus[locale].heading }}
+</h2>
+
+<p class="mb-400">
+	{{ helpus[locale].paragraph }}
+</p>
+
+<ul class="mb-600">
+	<li class="md:d-inline sm:d-block me-200">
+		<gcds-button
+			type="link"
+			button-role="primary"
+			href="{{ helpus[locale].feedbackHref }}"
+		>
+			{{ helpus[locale].feedback }}
+		</gcds-button>
+	</li>
+
+	<li class="md:d-inline sm:d-block mt-150 md:mt-0">
+		<gcds-button
+			type="link"
+			button-role="secondary"
+			target="_blank"
+			href="https://github.com/cds-snc/gcds-components"
+			lang="{{ locale }}"
+		>
+			{{ helpus[locale].github }}
+		</gcds-button>
+	</li>
+</ul>
```

Comment:
The spacing looks a bit wonky in this file, do you mind taking a look at this?

Reply:
I do not know why GitHub is displaying this way